app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -296,7 +291,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
     protected void onResume() {
         super.onResume();
 
-        updateLaunchButtonText(Pokefly.isRunning());
+        LocalBroadcastManager.getInstance(this).sendBroadcast(new Intent(Pokefly.ACTION_UPDATE_UI));
```

Comment:
I wondered if we still need `updateLaunchButtonText` for some path, but apparently (from testing) the button is now updated correctly anyway, even when resuming. 👍  @thearaks is there some scenario where this is needed?